sdk/core/azure-core/CHANGELOG.md

```diff
@@ -5,6 +5,9 @@
 ### Breaking Changes
 
 - `Azure::Core::Http::Url::AppendPath` now does not encode the input by default.
+- `azure.hpp` renamed to `strings.hpp`
```

Comment:
I think this should be:
```suggestion
- Removed `azure.hpp`.
```

And under the features heading:
- Added strings.hpp with the \<fill this in\> tolower and other methods.

Reply:
I'd use what Victor has since the type is moving from azure.hpp to strings.hpp

Reply:
Which type? All we have are private methods in there.